config/locales/application_states.yml

```diff
@@ -0,0 +1,9 @@
+en:
```

Comment:
Can we rename this file to be `candidate_application_states` or namespace somehow?
We have different state names for different users.